src/Files.App/Data/Items/DriveItem.cs

```diff
@@ -202,6 +202,14 @@ public object ToolTip
 		{
 			get
 			{
+				if (Section == SectionType.Favorites)
```

Comment:
What's this for?

Reply:
It's for showing the pinned icon.

Reply:
![1706136177063~2.png](https://github.com/files-community/Files/assets/49156174/63dc460e-bc15-4fec-92e9-c02c14214034)

Reply:
Isn't that already handled by the sidebar?

Reply:
I believe the sidebar only shows the icon defined by the pinned item. The other items have the icon because it is defined by `locationItem`.